proxylib/test/tmpdir.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2018 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package test
+
+import (
+	"io/ioutil"
+
+	log "github.com/sirupsen/logrus"
+)
+
+var Tmpdir string
```

Comment:
who will delete this directory when the tests are done? Consider using `c.MkDir()`

```
package main

import (
	. "gopkg.in/check.v1"
)

func (pl *ProxyLib) SetUpSuite(c *C) {
	Tmpdir := c.MkDir()
...
}
```

Reply:
```
go test ./... -check.work=true
```
```
-check.work - Display and do not remove the test working directory
```

It's up to you what you ;-)